snippets/csharp/System.Net/DownloadDataCompletedEventArgs/Overview/asyncmethods.cs

```diff
@@ -60,19 +60,31 @@ public static void UploadFileInBackground2(string address, string fileName)
             client.UploadFileCompleted += new UploadFileCompletedEventHandler(UploadFileCallback2);
 
             // Specify a progress notification handler.
-            client.UploadProgressChanged += new UploadProgressChangedEventHandler(UploadProgressCallback);
+            client.UploadProgressChanged += new UploadProgressChangedEventHandler(UploadProgressCallback2);
             client.UploadFileAsync(uri, "POST", fileName);
             Console.WriteLine("File upload started.");
         }
         //</Snippet4>
 
+        //<Snippet44>
```

Comment:
I think the snippet should not include the UploadFileCallback2 method to be consistent with examples for other event handlers

Reply:
UploadProgressChanged has two events subscribed. 
UploadFileCompleted has two event subscribed.
UploadValuesCompleted has zero events subscribed. This is on my list to fix.

If we want to make it consistent
UploadProgressChanged  should have one event (remove one)
UploadFileCompleted  should have one event (remove one)
UploadValuesCompleted  should have one event (add one)

Is this how we should handle these last three events?

Reply:
It can have multiple events subscribed to, IMO it does not have to show the definition of the other handlers. I.e. show only UploadFileCompleted handler definition in docs for UploadFileCompleted event, as in https://learn.microsoft.com/en-us/dotnet/api/system.net.webclient.uploadfilecompleted?view=net-6.0&branch=main

Reply:
@softwarepronto Do you plan to make any more changes here?